apps/emqx/etc/emqx.conf.rendered

```diff
@@ -0,0 +1,1671 @@
+##==================================================================
```

Comment:
Was this supposed to be commited?

I think this is generated by `emqx_common_test_helpers` when one uses `emqx_common_test_helpers:start_app`.

Reply:
✅